lte/protos/pipelined.proto

```diff
@@ -74,34 +74,45 @@ message RequestOriginType {
   OriginType type = 1;
 }
 
+message PolicyAndVersion {
+  PolicyRule rule = 1;
+  uint32 version = 2;
+}
+
 message ActivateFlowsRequest {
   SubscriberID sid = 1;
   string ip_addr = 2; // Subscriber session ipv4 address
-  repeated PolicyRule dynamic_rules = 4;  // List of dynamic rules obtained from PCRF
+  repeated PolicyRule dynamic_rules = 4;  // List of dynamic rules obtained from PCRF TODO deprecate
   RequestOriginType request_origin = 5; // Activate flow source (GX/GY)
   magma.lte.AggregatedMaximumBitrate apn_ambr = 6; // APN AMBR info, passed at the time of session creation
   bytes ipv6_addr = 7; // Subscriber session ipv6 address
   bytes msisdn = 8; // From SubscriberDB or HSS
   uint32 uplink_tunnel = 9; // Subscriber uplink tunnel
   uint32 downlink_tunnel = 10; // Subscriber downlink tunnel
-
+  repeated PolicyAndVersion rules = 11; // Policy + version pairs
   reserved 3;
 }
 
+message PolicyIDAndVersion {
```

Comment:
Already defined above? Also should we name this something like VersionedPolicy, having 'and' in the name sounds a bit off to me

Reply:
ah sounds good. I'll rename
PolicyAndVersion -> VersionedPolicy
PolicyIDAndVersion -> VersionedPolicyID ?